.github/workflows/upstream-issues.yml

```diff
@@ -0,0 +1,13 @@
+name: Upstream issue notifier
```

Comment:
still have to add the schedule to run it daily/weekly

Reply:
Yeah I just wanted your approval of it pre-pushing since if I say..... did the cron every minute by accident we'd be seeing a lot of updates, but pushing it now!